pyomo/repn/tests/mps/knapsack_problem_binary_variable_declaration.mps.baseline

```diff
@@ -0,0 +1,39 @@
+* Source:     Pyomo MPS Writer
+* Format:     Free MPS
+*
+NAME knapsack problem
+OBJSENSE
+ MIN
+ROWS
+ N  obj
+ G  c_l_const1_
+COLUMNS
+     INT 'MARKER' 'INTORG'
+     x(_1_) obj 3
+     x(_1_) c_l_const1_ 30
+     x(_2_) obj 2
+     x(_2_) c_l_const1_ 24
+     x(_3_) obj 2
+     x(_3_) c_l_const1_ 11
+     x(_4_) obj 4
+     x(_4_) c_l_const1_ 35
+     x(_5_) obj 5
+     x(_5_) c_l_const1_ 29
+     x(_6_) obj 4
+     x(_6_) c_l_const1_ 8
+     x(_7_) obj 3
+     x(_7_) c_l_const1_ 31
+     x(_8_) obj 1
+     x(_8_) c_l_const1_ 18
```

Comment:
Several solvers documentation indicate that INTEND is required.  The logic should be updated to ensure that the INTEND marker is always emitted if `set_integer` is True at the end of the COLUMNS section.